### ISSUE TITLE
"Empty snippet" is an invalid snippet

### ISSUE THREAD

**Issue by jalovatt** [Microsoft/vscode#53454]
Issue Type: <b>Bug</b>

- Open one of the user snippets JSON files
- Type " to bring up the empty snippet suggestion
- Tab to insert it

The provided snippet inserts top-level curly braces, which (I don't know JSON) an existing snippet file already has, making the new snippet invalid.

```JSON
// Valid
{
	"Existing Snippet": {
		"prefix": "my_snippet",
		"body": "this is a snippet",
		"description": "I'm an existing snippet"
	}
}

// "End of file expected" warning on the next line
{       
	"snippetName": {
		"prefix": "prefix",
		"body": "snippet",
		"description": "description"
	}
}
```

VS Code version: Code 1.24.1 (24f62626b222e9a8313213fb64b10d741a326288, 2018-06-13T17:51:32.889Z)
OS version: Windows_NT x64 6.3.9600

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-3630QM CPU @ 2.40GHz (8 x 2395)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.95GB (10.49GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-lua|gcc|0.1.2
color-highlight|nau|2.3.0

(1 theme extensions excluded)

</details>

**Comment by jrieken** [comment]
Yeah, I'd say the default proposal shouldn't show when the file is already populated. @aeschli Unsure if that is in my schema or not
